app/views/works/_work_form_tags.html.erb

```diff
@@ -1,9 +1,18 @@
 <fieldset class="work meta">
   <legend><%= ts('Tags') %></legend>
   <h3 class="landmark heading"><%= ts('Tags') %></h3>
+
   <p class="note"><%= ts('Tags are comma separated, %{max} characters per tag.', :max => ArchiveConfig.TAG_MAX) %></p>
 
   <dl>
+    <% import ||= false
```

Comment:
I've had another go at formatting the form by putting the notes into the main form which might improve the logical flow.